### ISSUE TITLE
Missing schema/search_path in PostgreSQL

### ISSUE THREAD

**Issue by dnovikov** [kitloong/laravel-migrations-generator#150]
Running `php artisan migrate:generate --tables="Customers"` on PostgreSQL leads to:

```
Doctrine\DBAL\Exception\TableNotFoundException 

  An exception occurred while executing 'SELECT obj_description('Customers'::regclass) AS table_comment;':

SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "customers" does not exist
 - Doctrine DBAL Version [`doctrine/dbal`]:
```

`SELECT obj_description('public."Customers"'::regclass) AS table_comment;` works fine.

**Comment by kitloong** [comment]
Hello @dnovikov , thanks for reporting.

Could you please provide the following?

**Details (please complete the following information):**
 - DB: [e.g. MySQL]
 - DB Version: [e.g. 5.7.37]
 - Laravel Version: 
 - PHP Version: 
 - Migrations Generator Version: 
 - Doctrine DBAL Version [`doctrine/dbal`]:

**Comment by dnovikov** [comment]
Hello @kitloong , sure. Just was in hurry when was creating this issue.

- PostgreSQL
- 11.13
- v7.30.6
- 7.3.23
- v6.6.3
- 2.13.9

**Comment by kitloong** [comment]
Hi @dnovikov , I am sorry for the late response.

This error is resolved with `doctrine/dbal:^3.0`.
I will close this issue for now. You should consider upgrading your project versions.
